wiki/Ranking_Criteria/osu!taiko/en.md

```diff
@@ -3,46 +3,161 @@ osu!taiko
+
+**Rhythm related guidelines and rules apply to approximately 180 BPM maps.** If your song is drastically faster or slower, some variables might be different. Apply reasonable judgment in these cases.
+
+If the beatmap follows a double or halved BPM style, Slider Velocity and all snapping variables in this ruleset need to be adjusted respectively.
+
+### ![Kantan](/wiki/shared/Easy-t.png "Kantan") Kantan
+
+#### Rules
+- **If a 1/2 pattern is used, the patterns must stay simplistic and be followed by a rest moment.** For songs that follow a swing beat, this limit is 1/3.
+- **Patterns on snaps faster than 1/2 are disallowed.** These patterns are too complex for beginner players. For songs which follow a swing beat, this limit is 1/3 instead.
+
+#### Guidelines
+- **1/1 patterns should not be longer than seven notes.** Anything longer is likely to be too straining for beginners. Patterns like these should be followed by a rest moment.
+- **Main snapping should consist of mostly 2/1, 4/1, or slower rhythms.** Occasionally using 1/1 rhythms is acceptable.
+- **Try to insert at least 1 rest moment that is 3/1 or longer after 16/1 to 20/1 of continuous mapping.** Less frequent rest moments or shorter ones may put too much strain on beginners.
+- **Slider velocity changes may be used cautiously.** Changes should only happen for sections of different pacings and slider velocity should not be drastically variable. 
+
+#### Difficulty Setting Guidelines
+- OD should be 4 or less.
+- HP should be 6 or more. In cases of higher note count, HP could be slightly adjusted to lower than 6.
+
+### ![Futsuu](/wiki/shared/Normal-t.png "Futsuu") Futsuu
+
+#### Rules
+- **Patterns on snaps faster than 1/3 are disallowed.** These patterns are too complex for beginner players.
+
+#### Guidelines
+- **1/3 patterns should not be longer than two notes.** Anything longer is very situational and usually too complex for newer players. Patterns like these should be followed by a rest moment.
+- **1/2  patterns should not be longer than seven notes.** Anything longer is likely to be too straining for beginners.
+- **Main snapping should consist of mostly 1/1, 2/1 or slower rhythms.** Occasionally using 1/2 rhythms is acceptable.
+- **Try to insert at least 1 rest moment that is 2/1 or longer after 16/1 to 20/1 of continuous mapping.** Less frequent rest moments or shorter ones may put too much strain on beginners.
+- **Simple Slider velocity changes may be used.**
+
+If you plan on using a Futsuu as the lowest difficulty of a mapset, it has to abide by the following rules and guidelines:
+
+#### Rules
+- **Patterns on snaps faster than 1/2 are disallowed.** These patterns are too complex for beginner players.
+
+#### Guidelines
+- **1/2 patterns should not be longer than four notes.**
+- **Slider velocity changes may be used cautiously.** Changes should only happen for sections of different pacings and slider velocity should not be drastically variable. 
```

Comment:
this should rather read "should not be used". It makes it clear that they are not directly disallowed, but still not recommended

Reply:
after player feedback, we discovered that kantan players do not suffer too much with mild sv changes so we wanted to loosen the criteria a bit here, hence the "may" instead of should not.